docs/DeveloperGuide.md

```diff
@@ -273,6 +273,83 @@ the gender is not of valid format; Invalid person exception is thrown if the per
 contact list. Error message is displayed on the GUI subsequently.
 </div>
 
+### Add Date Of Birth
+
+The Add Date Of Birth feature allows users to add a date of birth field (format: dd/mm/yyyy) to a person in the contact list. It is performed as a part of `addPersonCommand#execute()`.
+
+These operations are exposed in the `Model` interface as the method `Model#addPerson()`, which calls
+`AddressBook#addPerson()` which calls `UniquePersonList#add()` to add a new person in the person list
+stored in AddressBook.
+
+The following sequence diagram shows the methods calls related to add person operation:
+
+![AddPersonSequenceDiagram](images/AddPersonSequenceDiagram.png)
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** `cmd` in the diagram represents the add
+person command text entered by user. The specific `UniquePersonList` operations are not shown in the diagram
+for simplicity.
+</div>
+
+The following activity diagram shows what happens when a user executes a new add command:
```

Comment:
To standardize with line 284, we should call it - new add person command (we have yet to rename it but it'll be done this iteration so using the term 'add person' is preferable to 'add' in this DG)